### PR TITLE
Add documentation for Mercury

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

File: docs/core/Mercury.md
Comment (jmank88) [inline docs/core/Mercury.md:1]: Maybe this belongs up one level, in `docs/`?

File: docs/core/Mercury.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+          "resultPath": "data,mid"
+        }
+      },
+      "status": "testing"
+    }
+  },
+  "name": "Mercury v0.2 - Production Testnet Verifier (v1.0.0)",
+  "status": "testing"
+}
+```
```
Comment (jmank88) [inline docs/core/Mercury.md:158]: You could use `<details><summary>` sections to make examples collapsible. I don't know how to suggest with the nested `, but you should be able to 'edit' this comment to see the source I used:

<details><summary>Example `verifier/<0xaddress>.json`:</summary>

```json
{
  "contractVersion": 1001,
  "digests": {
    "0x0006c67c0374ab0dcfa45c63b37df2ea8d16fb903c043caa98065033e9c15666": {
      "feedId": "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c",
      "proxyEnabled": true,
      "status": "active"
    }
  },
  "feeds": {
    "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c": {
      "digests": [
        "0x0006c67c0374ab0dcfa45c63b37df2ea8d16fb903c043caa98065033e9c15666"
      ],
      "docs": {
        "assetName": "Chainlink",
        "feedCategory": "verified",
        "feedType": "Crypto",
        "hidden": true
      },
      "externalAdapterRequestParams": {
        "endpoint": "cryptolwba",
        "from": "LINK",
        "to": "USD"
      },
      "feedId": "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c",
      "latestConfig": {
        "offchainConfig": {
          "deltaGrace": "0",
          "deltaProgress": "2s",
          "deltaResend": "20s",
          "deltaRound": "250ms",
          "deltaStage": "60s",
          "f": 1,
          "maxDurationObservation": "250ms",
          "maxDurationQuery": "0s",
          "maxDurationReport": "250ms",
          "maxDurationShouldAcceptFinalizedReport": "250ms",
          "maxDurationShouldTransmitAcceptedReport": "250ms",
          "rMax": 100,
          "reportingPluginConfig": {
            "alphaAcceptInfinite": false,
            "alphaAcceptPpb": "0",
            "alphaReportInfinite": false,
            "alphaReportPpb": "0",
            "deltaC": "0s"
          },
          "s": [
            4
          ]
        },
        "offchainConfigVersion": 30,
        "onchainConfig": {
          "max": "99999999999999999999999999999",
          "min": "1"
        },
        "onchainConfigVersion": 1,
        "oracles": [
          {
            "api": [
              "coinmetrics",
              "ncfx",
              "tiingo-test"
            ],
            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-0"
          },
          {
            "api": [
              "coinmetrics",
              "ncfx",
              "tiingo-test"
            ],
            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-1"
          },
          {
            "api": [
              "coinmetrics",
              "ncfx",
              "tiingo-test"
            ],
            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-2"
          },
          {
            "api": [
              "coinmetrics",
              "ncfx",
              "tiingo-test"
            ],
            "operator": "clc-ocr-mercury-arbitrum-goerli-nodes-3"
          }
        ]
      },
      "marketing": {
        "category": "crypto",
        "history": true,
        "pair": [
          "LINK",
          "USD"
        ],
        "path": "link-usd-verifier"
      },
      "name": "LINK/USD-RefPricePlus-ArbitrumGoerli-002",
      "reportFields": {
        "ask": {
          "decimals": 8,
          "maxSubmissionValue": "99999999999999999999999999999",
          "minSubmissionValue": "1",
          "resultPath": "data,ask"
        },
        "bid": {
          "decimals": 8,
          "maxSubmissionValue": "99999999999999999999999999999",
          "minSubmissionValue": "1",
          "resultPath": "data,bid"
        },
        "median": {
          "decimals": 8,
          "maxSubmissionValue": "99999999999999999999999999999",
          "minSubmissionValue": "1",
          "resultPath": "data,mid"
        }
      },
      "status": "testing"
    }
  },
  "name": "Mercury v0.2 - Production Testnet Verifier (v1.0.0)",
  "status": "testing"
}
```
</details>

File: docs/Mercury.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+`f` - set this to `n//3` (where `//` denotes integer division), e.g. if you have 16 oracles, set `f` to 5.
+
+`deltaRound` - report generation frequency. This determines how frequently a new round should be started at most (if rounds take longer than this due to network latency, there will be fewer rounds per second than this parameter would suggest). `100ms` is a good starting point (10 rounds/s).
+
+`reportingPluginConfig.alphaAccept` - set this to `0`, because our mercury ContractTransmitter doesn't know the latest report that's been sent to the mercury server and we therefore always have a "pending" report which we compare against before accepting a report for transmission.
+
+`reportingPluginConfig.deltaC` - set this to `0` so every round will result in a report.
+
+<details><summary>Example `verifier/<0xaddress>.json`:</summary>
+```json
```
Comment (jmank88) [inline docs/Mercury.md:28]: For some reason, there must be a new line in between the html tags and the code block for these to render properly :shrug: 
```suggestion
<details><summary>Example `verifier/<0xaddress>.json`:</summary>

```json
```

**Review by jmank88 — APPROVED** [review]

File: docs/Mercury.md [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+
+[Reference contract](https://github.com/smartcontractkit/reference-data-directory/blob/master/ethereum-testnet-goerli-arbitrum-1/contracts/0x535051166466D159da8742167c9CA1eFe9e82613.json)
+
+[OCR2 config documentation](https://github.com/smartcontractkit/libocr/blob/master/offchainreporting2/internal/config/public_config.go)
+
+**🚨 Important config**
+
+`s` - transmission schedule. This should be set to the number of oracles on the feed - meaning that every oracle will attempt to transmit to the mercury server in the first stage of transmission. eg `[4]` if there are 4 node in the DON, excluding the bootstrap node.
+
+`f` - set this to `n//3` (where `//` denotes integer division), e.g. if you have 16 oracles, set `f` to 5.
```
Comment (kaleofduty) [inline docs/Mercury.md:19]: This should be `(n-1)//3`

File: docs/Mercury.md [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+          "maxDurationReport": "250ms",
+          "maxDurationShouldAcceptFinalizedReport": "250ms",
+          "maxDurationShouldTransmitAcceptedReport": "250ms",
+          "rMax": 100,
+          "reportingPluginConfig": {
+            "alphaAcceptInfinite": false,
+            "alphaAcceptPpb": "0",
+            "alphaReportInfinite": false,
+            "alphaReportPpb": "0",
+            "deltaC": "0s"
```
Comment (kaleofduty) [inline docs/Mercury.md:75]: could you link me to the place in the mercury plugin where these are used?

File: docs/Mercury.md [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+        "hidden": true
+      },
+      "externalAdapterRequestParams": {
+        "endpoint": "cryptolwba",
+        "from": "LINK",
+        "to": "USD"
+      },
+      "feedId": "0x14e044f932bb959cc2aa8dc1ba110c09224e639aae00264c1ffc2a0830904a3c",
+      "latestConfig": {
+        "offchainConfig": {
```
Comment (kaleofduty) [inline docs/Mercury.md:57]: where is deltaInitial?
This looks more like an OCR2 config.

File: docs/Mercury.md [hunk 6]
```diff
@@ -0,0 +1,350 @@
...
+          "rMax": 100,
+          "reportingPluginConfig": {
+            "alphaAcceptInfinite": false,
+            "alphaAcceptPpb": "0",
+            "alphaReportInfinite": false,
+            "alphaReportPpb": "0",
+            "deltaC": "0s"
+          },
+          "s": [
+            4
```
Comment (kaleofduty) [inline docs/Mercury.md:78]: has to be equal to number of nodes

File: docs/Mercury.md [hunk 7]
```diff
@@ -0,0 +1,350 @@
...
+
+Use this tool to configure contracts:
+
+https://github.com/smartcontractkit/the-most-amazing-mercury-contract-configuration-tool
+
+TODO: updated process here @Austin Born 
+
+[Reference contract](https://github.com/smartcontractkit/reference-data-directory/blob/master/ethereum-testnet-goerli-arbitrum-1/contracts/0x535051166466D159da8742167c9CA1eFe9e82613.json)
+
+[OCR2 config documentation](https://github.com/smartcontractkit/libocr/blob/master/offchainreporting2/internal/config/public_config.go)
```
Comment (kaleofduty) [inline docs/Mercury.md:13]: Why OCR2? We are using OCR3 for mercury.

File: docs/Mercury.md (hunk 5 above)
Comment (austinborn) [reply docs/Mercury.md:57]: I can confirm this is not what we are using for Data Streams jobs anymore. I think this doc was from a previous iteration when we were still on OCR2.

**Pull request merged**